parser/mpFuncStr.cpp

```diff
@@ -42,6 +42,37 @@
 
 MUP_NAMESPACE_START
 
+  //------------------------------------------------------------------------------
+  //
+  // Contains function
+  //
+  //------------------------------------------------------------------------------
+
+  FunStrContains::FunStrContains()
+    :ICallback(cmFUNC, _T("contains"), 2)
+  {}
+
+  //------------------------------------------------------------------------------
+  void FunStrContains::Eval(ptr_val_type &ret, const ptr_val_type *a_pArg, int)
+  {
+    const string_type & str1 = a_pArg[0]->GetString();
+    const string_type & str2 = a_pArg[1]->GetString();
+
+    *ret = str1.find(str2) != string_type::npos ? true : false;

```

Comment:
We can fix this unneeded ternary operator later on.

https://eslint.org/docs/rules/no-unneeded-ternary

<img width="939" alt="Screen Shot 2021-05-20 at 04 54 12" src="https://user-images.githubusercontent.com/7637806/118940797-743dc380-b927-11eb-94ea-d872e0533f51.png">